docs/source/tutorials/cosmos-transfer-integration.ipynb

```
@@ -0,0 +1,941 @@
+    "\n",
+    "Cosmos-Transfer works on **videos**.  \n",
+    "If your dataset consists of **images**, you must convert them into short MP4 video clips before using them as inputs.\n",
+    "\n",
+    "In this example, the FiftyOne dataset is downloaded from the HuggingFace Hub into the local FiftyOne directory, typically located at:\n",
+    "\n",
+    "```~/fiftyone/huggingface/hub/<username>/<dataset_name>/```\n",
+    "\n",
+    "Each sample image will be converted into a **1-second video** with the same filename.\n",
+    "\n",
+    "---\n",
+    "\n",
```

Ruff (0.14.7)

`typing.List` is deprecated, use `list` instead

(UP035)